src/server/scripts/Pandaria/zone_the_veiled_stair.cpp

```diff
@@ -24,7 +24,7 @@
 #define WRATHION_MAX_WP_Q4 7
 #define FEL_METEORS_RANGE 6.0f
 #define SHA_TOUCHED_GEM_RANGE 10.0f
-#define WRATHION_GOSSIP "Let`s talk..."
+#define WRATHION_GOSSIP "Давай поговорим..."
```

Comment:
Why?
Leave it in english

Reply:
Better to put gossips in the DB and use locals for other languages.